generators/common/index.js

```diff
@@ -54,9 +54,7 @@ module.exports = class extends BaseBlueprintGenerator {
     _initializing() {
         return {
             validateFromCli() {
-                if (!this.options['from-cli']) {
-                    this.error('This JHipster subgenerator is not intented for standalone use.');
-                }
+                this.checkInvocationFromCLI();
```

Comment:
This is not the same.

Reply:
Right. But see this commit: https://github.com/jhipster/generator-jhipster/pull/8959/commits/7678ce7be8e5b54537d49de7c30407464d5da7a3
Maybe I should have done 2 different PR for the 2 commits